insights/parsers/umask.py

```diff
@@ -0,0 +1,49 @@
+"""
+Umask - command ``/usr/bin/umask``
+==================================
+The ``Umask`` class implements the parsing of ``/usr/bin/umask``
+command, which is the value of default system umask.
+"""
+
+from insights import Parser, parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.umask)
+class Umask(Parser, dict):
+    """
+    Parser for ``/usr/bin/umask`` file.
+
+    Sample input is provided in the *Examples*.
+
+    Sample content::
+
+        0022
+
+    Examples:
+
+        >>> type(umask_obj)
+        <class 'insights.parsers.umask.Umask'>
+        >>> umask_obj['value']
+        '0022'
```

Comment:
A dictionary really doesn't make sense here, at least the way you have used it.  Why not use `umask -S` and create keys for `user`, `group` and `other`, and then have a tuple or list of the settings for that key?  The way it is currently implemented it simply a copy of the spec.

Reply:
@bfahr - Updated